pkg/k8s/apis/cilium.io/v2/cec_types.go

```diff
@@ -0,0 +1,115 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2021 Authors of Cilium
+
+package v2
```

Comment:
It's a new API, why is it v2 already?

Reply:
I had serious trouble with `v2alpha1` but it turned out to be an issue with the only existing `v2alpha1` API I was patterning my work on.. will switch back to `v2alpha1` later, assuming that would be the right thing to do.